host_core/mix.exs

```diff
@@ -33,15 +33,16 @@ defmodule HostCore.MixProject do
       {:wasmex, "~> 0.5.0"},
       {:msgpax, "~> 2.3"},
       {:rustler, "~> 0.22.0"},
+      {:jason, "~> 1.2.2"},
       {:gnat, "~> 1.2"},
       {:cloudevents, "~> 0.4.0"},
       {:uuid, "~> 1.1"},
-      {:jason, "~> 1.2.2"},
+
```

Comment:
Good for Jason, moving its way up the dependency ladder

Reply:
Everybody loves an underdog story